packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -202,14 +193,32 @@ export const BasicMultiCombobox: React.FC = () => {
     getToggleButtonProps,
     highlightedIndex,
     isOpen,
-    selectedItem,
+    selectItem,
   } = useComboboxPrimitive({
-    items: filteredItems,
-    onSelectedItemChange: (args) => {
-      handleSelectItemOnClick(args);
+    inputValue,
+    items: getFilteredItems(),
+    onStateChange: (args) => {
+      switch (args.type) {
+        case useComboboxPrimitive.stateChangeTypes.InputChange:
+          setInputValue(args.inputValue);
+          break;
+        case useComboboxPrimitive.stateChangeTypes.InputKeyDownEnter:
+        case useComboboxPrimitive.stateChangeTypes.ItemClick:
+        case useComboboxPrimitive.stateChangeTypes.InputBlur:
+          if (args.selectedItem) {
+            addSelectedItem(args.selectedItem);
+            setInputValue('');
+            selectItem(null);
```

Comment:
This is what caused the bug!!!! The selected item stayed the same, so when it was re-added it downshift ignored it.